cookbook/security/remember_me.rst

```diff
@@ -167,15 +173,18 @@ The Security component provides an easy way to do this. In addition to roles
 explicitly assigned to them, users are automatically given one of the following
 roles depending on how they are authenticated:
 
-* ``IS_AUTHENTICATED_ANONYMOUSLY`` - automatically assigned to a user who is
-  in a firewall protected part of the site but who has not actually logged in.
-  This is only possible if anonymous access has been allowed.
+``IS_AUTHENTICATED_ANONYMOUSLY``
+  Automatically assigned to a user who is in a firewall protected part of the
+  site but who has not actually logged in. This is only possible if anonymous
+  access has been allowed.
 
-* ``IS_AUTHENTICATED_REMEMBERED`` - automatically assigned to a user who
-  was authenticated via a remember me cookie.
+``IS_AUTHENTICATED_REMEMBERED``
+  Automatically assigned to a user who was authenticated via a remember me
```

Comment:
As pointed previously by @xabbuh, the description should be indented if we want to display it as a definition list.